content/docs/user-guide/components/reference/physx/rigid-body.md

```diff
@@ -0,0 +1,54 @@
+---
+linkTitle: PhysX Rigid Body
+title: PhysX Rigid Body Component
+description: The PhysX Rigid Body component creates a dynamic or kinematic solid object that can move and collide with other PhysX entities.
+toc: true
+---
+
+The **PhysX Rigid Body** component makes an entity a *dynamic* or *kinematic* solid object that can move and collide with other PhysX entities. The entity must also have at least one [PhysX Shape Collider](/docs/user-guide/components/reference/physx/shape-collider/)  or [PhysX Collider](/docs/user-guide/components/reference/physx/collider/)  component that defines a collider for the entity.
+
+Dynamic rigid bodies are fully simulated by PhysX. Dynamic rigid bodies move in response to collision events and forces and are not animated through motions or scripts. By default, dynamic rigid bodies are affected by gravity, but gravity can be deactivated in the PhysX Rigid Body component.
```

Comment:
```suggestion
Dynamic rigid bodies are fully simulated by [NVIDIA PhysX](https://developer.nvidia.com/physx-sdk). Dynamic rigid bodies move in response to collision events and forces and are not animated through motions or scripts. By default, dynamic rigid bodies are affected by gravity, but gravity can be deactivated in the PhysX Rigid Body component.
```

Reply:
This is pretty deep in the documentation. I don't think the link to NVIDIA is necessary here. We do link to it at much higher levels of the documentation.